websites/K/Kickassanime/presence.ts

```diff
@@ -56,6 +56,16 @@ function checkIfMovie() {
     : nextEpisodeElement !== null && previousEpisodeElement !== null
     ? (isMovie = false)
     : (isMovie = true);
+
+  !isMovie
+    ? presence
+        .getPageletiable("appData")
+        .catch((appData) =>
```

Comment:
eee catch? so you catch error then check anime name? I really don't understand it, could you explain it? Thanks.

Reply:
Wait. I think I might have fucked this XD

Reply:
P.S. ``.then`` instead of ``.catch``